docs/element-definitions.md

```diff
@@ -1715,21 +2124,30 @@ This `extension` element has a custom content, and the meaning and handling inst
 
 ```json
 {
-    "element": "extension",
-    "meta": {
-        "links": [
-            {
-                "element": "link",
-                "attributes": {
-                  "relation": "profile",
-                  "href": "http://example.com/extensions/info/"
-                }
+  "element": "extension",
+  "meta": {
+    "links": {
+      "element": "array",
+      "content": [
+        {
+          "element": "link",
+          "attributes": {
+            "relation": {
+              "element": "string",
+              "content": "profile"
+            },
+            "href": {
+              "element": "string",
+              "content": "http://example.com/extensions/info/"
             }
-        ]
-    },
-    "content": {
-      "version": "1.0"
+          }
+        }
+      ]
     }
+  },
+  "content": {
+    "version": "1.0"
```

Comment:
I am not sure what we need to do here. An example swagger document and refract is https://github.com/apiaryio/swagger-zoo/blob/master/fixtures/features/api-elements/extensions.json